src/ImageSharp/Processing/Processors/Transforms/EntropyCropProcessor.cs

```diff
@@ -43,14 +43,14 @@ public EntropyCropProcessor(float threshold)
         public float Threshold { get; }
 
         /// <inheritdoc/>
-        protected override void BeforeImageApply(Image<TPixel> source, Rectangle sourceRectangle)
+        protected override void BeforeImageApply(Image<TPixel> source, Rectangle sourceRectangle, Configuration configuration)
         {
             Rectangle rectangle;
 
             // All frames have be the same size so we only need to calculate the correct dimensions for the first frame
             using (ImageFrame<TPixel> temp = source.Frames.RootFrame.Clone())
             {
-                Configuration configuration = source.GetConfiguration();
+                configuration = configuration ?? source.GetConfiguration();
```

Comment:
Obtaining the `Configuration` instance should not be the responsibility of individual processors. This is error prone!

The `Configuration` instance we consume in this method should not be `null`, but the instance we expect in this context.